app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,10 @@
     -->
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
     <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
```

Comment:
Yeah that's true, but if they're just uploading a picture and not taking it they only need to be able to read so we only ask for write permissions when they're actually taking the picture